### ISSUE TITLE
Errors loading custom text

### ISSUE THREAD

**Issue by xjamesaaron** [cslarsen/wpm#37]
While loading a custom txt file, but formatted in C programming language, spaces like 'new line' and  'tabs' are not calculated properly and will not work normally.

**Comment by cslarsen** [comment]
Yep, it translates a lot of stuff so won't work well with things like programming languages. I'm swamped with a lot of other stuff, so I won't have time to improve the situation myself.

**Comment by scopalaffairs** [comment]
@cslarsen could you merge the PR https://github.com/cslarsen/wpm/pull/56 @skvrahul opened?
That would close some duplicates here and boost the enjoyment of using this cool tool
